api/tools.ts

```
@@ -1,71 +1,17 @@
-            {
-              type: "text",
-              text: "Failed to join meeting",
-            },
-          ],
-          isError: true,
-        };
-      }
-    }
-  );
+  // Register bot tools
+  server = registerJoinTool(server, baasClient);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)